CPAC/resources/configs/pipeline_config_blank.yml

```diff
@@ -197,6 +197,8 @@ pipeline_setup:
 # PREPROCESSING
 # -------------
 surface_analysis:
+  abcd_prefreesurfer_prep:
+    run: Off
```

Comment:
Can you add a comment that describes what this key does?

Reply:
Oh, weird. I [did add a comment](https://github.com/FCP-INDI/C-PAC/pull/1932/commits/fb086fef5d0400e3f693cfcd2e7b8e1b34fa7495#diff-1176dc8cb1898141a01c164e6dcd422e91306dc5cdac6c3e8a2d087682eaeb6fR205), but it looks like the [theo bot account deleted it](https://github.com/FCP-INDI/C-PAC/pull/1932/commits/807ed70d11d775fa058d1baa5f52bd5841f3c5be)

Reply:
Seems like the bot pulls comments [from default not from blank](https://github.com/FCP-INDI/C-PAC/blob/73f7af268a06a581f42ada0e36a9540d38c89554/CPAC/utils/configuration/yaml_template.py#L470) (what I would have guessed as well) so I think if you add the comment to default it should work

Reply:
okay, I added it to all the appropriate pipeline configs. I also think that the bot _should_ be pulling from blank instead of default. So I think thats a bug and we should write up an issue for it.